python/mqt/syrec/simulation_view/workers/cancellable_worker_variants.py

```
@@ -0,0 +1,179 @@
+        - batchCompleted(float): The worker has produced a new batch of items, the runtime (in seconds) to produce a new batch is passed as the argument of the signal.
+
+    Inputs:
+        SendQueueElemType: The workers send queue element type.
+
+    Attributes:
+        send_queue: A thread-safe FIFO queue storing the elements produced by the worker. The queue is expected to be unbounded.
+    """
+
+    finished = QtCore.pyqtSignal(bool)
+    failed = QtCore.pyqtSignal(Exception)
+    batchCompleted = QtCore.pyqtSignal(float)  # noqa: N815
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `N815`)

Remove unused `noqa` directive

(RUF100)